tests/components/thumbnail-container-test.js

```diff
@@ -0,0 +1,205 @@
+            "width":120,
+            "height":80
+          }
+        },
+        "20":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_020.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "30":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_030.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "40":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_040.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "50":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_050.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "60":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_060.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "70":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_070.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "80":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_080.jpg",
+            "width":320,
+            "height":160
+          }
+        },
+        "90":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_090.jpg",
+            "width":120,
+            "height":80
+          }
+        },
+        "100":{
+          "120":{
+            "url":"http://media.video-cdn.espn.com/motion/Miercoles_100.jpg",
+            "width":120,
+            "height":80
+          }
+        }
+      },
+      "errors":[
+        {
+          "status":404,
+          "code":"Not Found",
+          "title":"unable to find thumbnail images",
+          "detail":"embed code not found"
+        }
+      ]
+    }
+  };
+
+  it('for isCarousel = false need to show thumbnails', function () {
+    var DOM = TestUtils.renderIntoDocument
+    (
+      <ThumbnailContainer
+        thumbnails={thumbnails}
+        isCarousel={false}
+        hoverPosition={80}
+        duration={100}
+        hoverTime={80}
+        scrubberBarWidth={100}
+        vrViewingDirection={{yaw: 0, roll: 0, pitch: 0}}
+        videoVr={true}
+        fullscreen={false}
+      />
+    );
+    var thumbnailContainer = TestUtils.findRenderedDOMComponentWithClass(DOM, 'oo-scrubber-thumbnail-wrapper');
+    var thumbnail = TestUtils.findRenderedDOMComponentWithClass(DOM, 'oo-scrubber-thumbnail-container');
+    expect(thumbnail).not.toBeNull();
+  });
+  it('for isCarousel = true need to show thumbnails', function () {
+    var DOM = TestUtils.renderIntoDocument
+    (
+      <ThumbnailContainer
+        thumbnails={thumbnails}
+        isCarousel={true}
+        hoverPosition={80}
```

Comment:
We are using this values in few places. Can we create constants for hover, duration, hoverTime, scrubberBarWidth